lib/mockbuilder.py

```diff
@@ -145,18 +145,23 @@ def _create_build_directory(self, package):
     def _destroy_build_directory(self):
         shutil.rmtree(self.build_dir)
 
-    def _save_rpm(self, package):
-        if not os.path.exists(package.build_results_dir):
-            LOG.info("Creating directory to store RPMs at %s " %
-                     package.build_results_dir)
-            os.makedirs(package.build_results_dir)
-
-        LOG.info("%s: Saving RPMs at %s"
-                 % (package.name, package.build_results_dir))
-        for f in os.listdir(self.build_dir):
-            if f.endswith(".rpm") and not f.endswith(".src.rpm"):
-                LOG.info("Saving %s at result directory %s" % (
-                    f, package.build_results_dir))
-                orig = os.path.join(self.build_dir, f)
-                dest = os.path.join(package.build_results_dir, f)
-                shutil.move(orig, dest)
+    def _copy_rpms(self, source_dir, target_dir):
+        """
+        Copy the RPMs created by building a package to a target directory.
+
+        Args:
+            source_dir(str): path to the source directory containing the
+                RPMs
+            target_dir(str): path to the target directory
+        """
+        if not os.path.exists(target_dir):
+            LOG.debug("Creating directory to store RPMs at %s " % target_dir)
+            os.makedirs(target_dir)
+
+        LOG.info("Copying RPMs from %s to %s" % (source_dir, target_dir))
+        for rpm_file in os.listdir(source_dir):
+            if rpm_file.endswith(".rpm") and not rpm_file.endswith(".src.rpm"):
+                LOG.info("Copying RPM file: %s" % rpm_file)
+                orig = os.path.join(source_dir, rpm_file)
+                dest = os.path.join(target_dir, rpm_file)
+                shutil.copy(orig, dest)
```

Comment:
It is not clear to me why you are copying instead of moving. Could you explain better? 

Minor: I'd prefer to move this functional change to a separate patch and leave only refactoring here.